feat(route53): latency based routing (#28723)
aws/aws-cdk: packages/aws-cdk-lib/aws-route53/README.md

  zone: myZone,
  target: route53.RecordTarget.fromIpAddresses('1.2.3.4'),
  weight: 10,
  setIdentifier: 'weighted-record-id',
});
```

> **Warning** It is not possible to specify `setIdentifier` in a simple routing without one of `weight`, `region` or `geoLocation` defined.

Reviewer: ```suggestion
**Warning** It is not possible to specify `setIdentifier` for a simple routing policy.
```
